deployments/deploy-ssh.sh

```diff
@@ -315,7 +325,8 @@ spec:
             "--custom-config", "$CUSTOM_APP_CONFIG_PATH",
             "--storage-upgrade-endpoint-url=$UPGRADE_ENDPOINT_URL",
             "--storage-upgrade-endpoint-version=$UPGRADE_ENDPOINT_VERSION",
-            "$APP_DESTROY_TIMEOUT_ARG"
+            "$APP_DESTROY_TIMEOUT_ARG",
+            "--backup-driver", "$BACKUP_DRIVER"
```

Comment:
Can you confirm this doesn't throw an error if the driver name is empty

Reply:
Run with "--backup-driver pxb"
Backup-Enabled: https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/352/. 

Run with driver name empty
Backup-Disabled: https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/353/